packages/ui-kit/src/Panel/index.tsx

```diff
@@ -44,33 +70,105 @@ export function Panel({
   contentContainerRef,
   contentClassName,
   resize = 'none',
+  shortStateContent,
+  isShortStateOpen,
+  shortStateListeners,
   ...rest
 }: React.PropsWithChildren<Panel>) {
 
+  function panelContent() {
+    if (isShortStateOpen && shortStateContent && isOpen) return (
+      <div
```

Comment:
and one possible moment to improve - memoize (i know it was before, but maybe you'll have a time)
`{ minHeight: minContentHeightPx || 'unset', resize: resize }`
it will create new object on every render